tests/models/test_models.py

```diff
@@ -0,0 +1,225 @@
+"""
+Test model
```

Comment:
The filename `test_models.py` and this comment are misleading. Can you rename it to something to do with saving and loading from checkpoints? `test_models.py` implies the models are being tested. I know this deviates slightly from our convention but I think it'll be more clear in this case.

Reply:
No problem I can change it, maybe to `test_load_model.py`? What do you think? For now I followed the convention on `test_vanilla_pipeline.py` which tests that the pipeline can be loaded similarly to this test :)